README.md

```diff
@@ -132,8 +130,13 @@ docker run -v /tmp:/tmp -e API_KEY=changeme ghcr.io/bolt-observer/agent:v0.0.35
 
 ## Filtering on agent side
 
-You can limit what channnels are reported using `--channels-whitelist` option. It specifies a path of a local file to be used as a whitelist of what channels to report.
-`--channels-whitelist` implies `--private` (and specifying both is a mistake). The format of the file is simple:
+You can limit what channnels are reported using `--channel-whitelist` option. It specifies a local file path to be used as a whitelist of what channels to report.
+When adding `--private` to `--channel-whitelist` this means every private channel AND whatever is listed in the file. There is also `--public` to allow all public channels.
+Using the options without `--channel-whitelist` makes no sense since by default all public channels are reported however it has to be explicit with `--channel-whitelist` in order
+to automatically allow all public channels (beside what is allowed through the file).
+If you set `--private` and `--public` then no matter what you add to the `--channel-whitelist` file everything will be reported.
+
+The format of the file is simple:
```

Comment:
remove simple

Reply:
this is not format but example.

Format would be something like `[pubkey | short id]` per line

Reply:
👍